.github/workflows/release.yml

```diff
@@ -82,14 +127,23 @@ jobs:
         id: extract_branch
         run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
       - name: Invoke Oclif workflow
-        if: ${{needs.release.outputs.tag-name != ''}}
+        if: ${{needs.release.outputs.tag-name != '' && github.event.inputs.draft-release != 'true'}}
         run: node .github/scripts/trigger-workflow.js
         env:
           WORKFLOW_NAME: 'oclif-release.yml'
           REPO_NAME: twilio/twilio-cli
           REPO_ACCESS_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
           BRANCH_NAME: ${{steps.extract_branch.outputs.branch}}
           INPUTS: '{ "home-brew-branch": "${{github.event.inputs.homebrew-branch}}", "tag-name": "${{needs.release.outputs.tag-name}}", "pre-release": "${{github.event.inputs.homebrew-prerelease}}"}'
+      - name: Invoke Oclif workflow for draft release
```

Comment:
RPM workflow will be triggered when cli release workflow is completed, so How are we not triggering RPM workflow in case of draft-release?

Reply:
We would, as a part of [DII-183](https://issues.corp.twilio.com/browse/DII-183). There'd be a separate PR for that.

Reply:
Yes, but my concern is tomorrow when you do the draft-release, RPM release will be triggered because it is dependent on cli release workflow. We need to add some condition in RPM workflow so that it will not be triggered when dradt-release triggered.